src/styles/global.css

```
@@ -0,0 +1,56 @@
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: var(--bg-subtle);
+  color: var(--text);
+  font-family:
+    "Datatype",
```

Stylelint (17.12.0)

[error] Expected no quotes around "Datatype" (font-family-name-quotes)

(font-family-name-quotes)